src/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react'
+import { Box, Typography } from '@mui/material'
+import QuizQuestion from '~/containers/my-quizzes//scroll-question/ScrollQuestions'
+
+import { QuizViewProps } from '~/types'
+import { styles } from '~/containers/my-quizzes/scroll-question-quiz-view/ScrollQuestionsQuizView.styles'
+
+const ScrollQuestionsQuizView: FC<QuizViewProps> = ({ questions }) => {
+  const questionList = questions.map((question, index) => (
+    <QuizQuestion index={index} key={question._id} question={question} />
+  ))
```

Comment:
If there are no questions? What will be with view tabs? Maybe you should add some checks here or on 20 lines when you render it.

Reply:
> If there are no questions? What will be with view tabs? Maybe you should add some checks here or on 20 lines when you render it.

FYI, now View tab disabled if we don't have any questions, your suggestion is not correct..